contrail-ui/src/components/main/main-app-bar/MainAppBar.tsx

```diff
@@ -0,0 +1,28 @@
+import AppBar from "@material-ui/core/AppBar";
+import CssBaseline from "@material-ui/core/CssBaseline";
+import Toolbar from "@material-ui/core/Toolbar";
+import Typography from "@material-ui/core/Typography";
+import { withStyles } from "@material-ui/styles";
+import React, { Component } from "react";
+import styles from "../mainStyles";
+
+class MainAppBar extends Component<any, any> {
+    public render() {
+        const { classes } = this.props;
+        return (
+            <body>
+                <div className={classes.root}>
```

Comment:
Remove body tag. I think we already have our body tag. If I'm wrong keep it

Reply:
Fixed